src/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+  margin: 0;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+}
+
+.bigSearch {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-right: 72px;
+  margin-left: 8px;
+}
+
+.smallSearch {
+  margin-top: 20px;
+  margin-bottom: 10px;
+  margin-right: 43px;
+  margin-left: 8px;
+}
+
+.input {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 66px;
+  padding-left: 62px;
+}
+
+.input--small {
+  width: 100%;
+  height: 38px;
+  padding-left: 33px;
+}

```

Comment:
create class input and apply to it all common styles for 2 inputs and create two external classes input--big and input--small and apply to them other specific styles for each input

Reply:
I created new class and changed styles.